icu4j/main/classes/core/src/com/ibm/icu/impl/personname/PersonNameFormatterImpl.java

```diff
@@ -193,6 +202,8 @@ public boolean shouldCapitalizeSurname() {
 
     private final Set<String> LOCALES_THAT_DONT_USE_SPACES = new HashSet<>(Arrays.asList("ja", "zh", "yue", "km", "lo", "my"));
```

Comment:
I don't think these are needed anymore. Not a blocker for this PR, but lets talk if your code still uses them, because it shouldn't need to.

Reply:
I have another ticket to clean up the foreignSpaceReplacement/nativeSpaceReplacement stuff, and I think that'll come out as part of that (I'll try to remember to double-check this).